public/components/ResultGroup.vue

```diff
@@ -332,7 +332,7 @@ export default Vue.extend({
 
     hasExplanations(): boolean {
       // waiting for explanation enum to be added to results summaries
-      return false;
+      return this.predictedSummary?.weighted;
```

Comment:
With [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) you can get a `undefined`, adding `!!` in front will make sure you get a boolean.

Reply:
Good point will update.